mne/io/utils.py

```diff
@@ -167,3 +170,43 @@ def _read_segments_file(raw, data, idx, fi, start, stop, cals, mult,
                 block = np.vstack((block, stim_ch))
             data_view = data[:, sample_start:sample_stop]
             _mult_cal_one(data_view, block, idx, cals, mult)
+
+
+def read_str(fid, count=1):
+    """Read string from a binary file in a version compatible way."""
+    dtype = np.dtype('>S%i' % count)
+    string = fid.read(dtype.itemsize)
+    data = np.fromstring(string, dtype=dtype)[0]
+    bytestr = b('').join([data[0:data.index(b('\x00')) if
+                          b('\x00') in data else count]])
+
+    return str(bytestr.decode('ascii'))  # Return native str type for Py2/3
+
+
+def _create_chs(ch_names, cals, ch_coil, ch_kind, eog, ecg, emg, misc):
```

Comment:
I took a quick look and it should be also used in egi and brainvision. There is almost the same code in both.

now ok if you want to do this in the next PR. But let's not forget.